### PR TITLE
Add contribution page w/ guidelines

### PR DESCRIPTION
**Pull request by srcarrier**
@dixonwhitmire let me know what you think.
Some of the content was inspired from the IBM FHIR team [contribution](https://github.com/IBM/FHIR/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Review by dixonwhitmire — APPROVED** [review]
@srcarrier This looks great! Thank you.
@ccorley Do you have any feedback on the our initial contribution page?

File: source/develop/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Contribute
+**********
+
+Contributions of all kinds are welcome and encouraged.
+
+To contribute code or documentation, please open a `pull request <https://github.com/LinuxForHealth/connect/pulls>`_.
+
```
Comment (ccorley) [inline source/develop/contribute.rst:7]: How about "To contribute code or documentation for an existing issue, please open a pull request"?  Then for new features and issues, you mention opening an issue first, which is perfect.

**Review by ccorley — APPROVED** [review]
Looks great.  Approving, with one minor clarifying comment.  I think we will always want an issue first, before a PR, unless there is already an existing issue.

**Review by ccorley — APPROVED** [review]
Looks great!

**Comment by torimcc** [comment]
Closing this pull request for now - as it covers the same documentation as the other Contributing doc I put together in the Connect repo. Thanks for your work!

**Pull request closed without merging**